File: MODUL_4/C/PRAK405_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include <stdio.h>

int main() {
    int n, Kelipatan;
    int Baris = 0;

    printf("Masukkan nilai n dan kelipatan : ");
    scanf("%d %d", &n, &Kelipatan);

    for (int i = 1; i <= n; i++) {
        int Totalbaris= 0;

        for (int j = i; j > 0; j--) {
            Totalbaris += j * Kelipatan;

            printf("(%d * %d)", j, Kelipatan);
            if (j > 1) {
                printf(" + ");
            }
        }

        printf(" = %d\n", Totalbaris);
        Baris += Totalbaris;
    }

    printf("%d\n", Baris);

    return 0;
}

File: MODUL_4/Py/PRAK405_2310817310014_MUHAMMADRIZKISAPUTRA.py
n, Kelipatan = map(int, input("Masukkan nilai n dan kelipatan: ").split())
Baris = 0

for i in range(1, n+1):
    Totalbaris = 0

    for j in range(i, 0, -1):
        Totalbaris += j * Kelipatan

        print(f"({j} * {Kelipatan})", end="")
        if j > 1:
            print(" + ", end="")

    print(f" = {Totalbaris}")
    Baris += Totalbaris

print(Baris)
